clients/cli/tests/helpers.rs

```diff
@@ -75,36 +62,227 @@ pub async fn setup() -> Env {
+            &unwrapped_token_program.to_string(),
+            &wrapped_token_program.to_string(),
+            "--idempotent",
+        ])
+        .status()
+        .unwrap();
+    assert!(status.success());
+}
+
+#[allow(clippy::too_many_arguments)]
+pub async fn execute_wrap(
+    env: &TestEnv,
+    unwrapped_token_program: &Pubkey,
+    unwrapped_token_account: &Pubkey,
+    escrow_account: &Pubkey,
+    wrapped_token_program: &Pubkey,
+    amount: u64,
+    mint_address: Option<&Pubkey>,
+    recipient_account: Option<&Pubkey>,
+) {
+    let mut args = vec![
+        "wrap".to_string(),
+        "-C".to_string(),
+        env.config_file_path.clone(),
+        unwrapped_token_program.to_string(),
+        unwrapped_token_account.to_string(),
+        escrow_account.to_string(),
+        wrapped_token_program.to_string(),
+        amount.to_string(),
+    ];
+
+    if let Some(mint) = mint_address {
+        args.push("--unwrapped-mint".to_string());
+        args.push(mint.to_string());
+    }
+
+    if let Some(recipient) = recipient_account {
+        args.push("--recipient-token-account".to_string());
+        args.push(recipient.to_string());
+    }
+
+    let status = Command::new(TOKEN_WRAP_CLI_BIN)
+        .args(args)
+        .status()
+        .unwrap();
+    assert!(status.success());
+}
+
+pub async fn create_associated_token_account(
+    env: &TestEnv,
+    token_program: &Pubkey,
+    mint: &Pubkey,
+) -> Pubkey {
+    let ata =
+        get_associated_token_address_with_program_id(&env.payer.pubkey(), mint, token_program);
+
+    let ata_account = env.rpc_client.get_account(&ata).await;
+    if ata_account.is_ok() {
+        return ata; // Return early if it exists
+    }
+
+    let instruction = spl_associated_token_account::instruction::create_associated_token_account(
+        &env.payer.pubkey(),
+        &env.payer.pubkey(),
+        mint,
+        token_program,
+    );
+
+    let tx = Transaction::new_signed_with_payer(
+        &[instruction],
+        Some(&env.payer.pubkey()),
+        &[&env.payer],
+        env.rpc_client.get_latest_blockhash().await.unwrap(),
+    );
+
+    env.rpc_client
+        .send_and_confirm_transaction(&tx)
+        .await
+        .unwrap();
+
+    ata
+}
+
+pub async fn create_token_account(
+    env: &TestEnv,
+    token_program: &Pubkey,
+    mint: &Pubkey,
+    owner: &Pubkey,
+) -> Pubkey {
+    let token_account = Keypair::new();
+    let account_size = spl_token::state::Account::LEN;
+
+    let initialize_instruction = if *token_program == spl_token::id() {
+        spl_token::instruction::initialize_account(
+            token_program,
+            &token_account.pubkey(),
+            mint,
+            owner,
+        )
+        .unwrap()
+    } else {
+        spl_token_2022::instruction::initialize_account(
+            token_program,
+            &token_account.pubkey(),
+            mint,
+            owner,
+        )
+        .unwrap()
```

Comment:
nit: the spl_token_2022 constructor works for both programs, so simpler just to use that
```suggestion
    let initialize_instruction = spl_token_2022::instruction::initialize_account(
            token_program,
            &token_account.pubkey(),
            mint,
            owner,
        )
        .unwrap()
```